helm-charts/yunikorn/templates/deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           lifecycle:
               postStart:
                 exec:
-                  command: ["/bin/sh", "/admission_util.sh", "create"]
+                  command: ["/bin/sh", "/admission_util.sh", "create", "--env", "NAMESPACE={{ .Release.Namespace }}"]
```

Comment:
Have you tested the case when we do helm install without giving a namespace?
The charts will be installed to default namespace, but I am not sure `Release.Namespace` will have the proper value set.

Reply:
I have tested this via calling command `helm install ./yunikorn --generate-name` , the result seems same as the command with `--namespace default`.

Reply:
Cool, thx